docs/configuration.md

```diff
@@ -26,6 +26,7 @@ The following table lists the parameters in the configuration file:
 | `plugin.base_path`                       | The folder to store plugins.                                                     | `${AppBaseDir}/plugins`                                                                |
 | `planner.example_base_path`              | The folder to store planner examples.                                            | `${AppBaseDir}/planner_examples`                                                       |
 | `planner.prompt_compression`             | Whether to compress the chat history for planner                                 | `false`                                                                                | 
+| `planner.skip_planning`                  | Whether to skip planning process.                                                | `false`                                                                                | 
```

Comment:
what is the relation between skip_planning and use_planner?

Reply:
We'd better have a dedicated doc for this topic.

Reply:
skip_planning means whether use a dummy plan to bypass the planning process.
use_planner means whether use the Planner component.
We need a detailed doc to describe both settings.